java/code/src/com/redhat/rhn/domain/channel/PublicChannelFamily.java

```diff
@@ -53,6 +72,7 @@ public boolean equals(final Object other) {
         }
         PublicChannelFamily otherFamily = (PublicChannelFamily) other;
         return new EqualsBuilder()
+            .append(getId(), otherFamily.getId())
```

Comment:
Not sure if the "id" should be in the equals and hashcode method. I remember that this is a bad idea.
@moio knows more.

Reply:
See my other comment on the equals. I will fix this.